### PR TITLE
Finish views

### PR DESCRIPTION
**Pull request by bienvenuushindi**
- Add the Views associated with the Models and Controllers 
- Implement the design from the sneak peek wireframes. 

| get users | get users/:id | get users/:id/posts | get users/:id/posts/:post_id |
| :---: |:---: |:---: |:---: |
|page1|page2|page3|page4

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <ul id="users" >
+    <% @users.each do |user| %>
+      <li class="user-item">
+        <div class="card row">
+          <%= image_tag(user.photo, class: "portrait") %>
+          <div class="d-flex flex-column justify-space-between w-100">
+            <h2 class="header text-start"><%= user.name %></h2>
+            <p class="text-end text-light">Number of posts: <%= user.posts_counter %></p>
+          </div>
+        </div>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/index.html.erb:13]: - [ ] Nice job @bienvenuushindi , making this user friendly kindly make sure that when a user card is being clicked, it directs to user/show

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <div class="d-flex gap-1 justify-content-end">
+            <div>Comments <span><%= post.comments_counter %></span></div>
+            <div>Likes <span><%= post.likes_counter %></span></div>
+          </div>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <div class="text-cent mt-5">
+    <button class="btn btn-default mx-auto">See All Project</button>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/show.html.erb:31]: - [ ] Also a click on this button link opens user's post index. Kindly fix that

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @bienvenuushindi  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @bienvenuushindi,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**